website/content/release-notes/_index.md

```diff
@@ -3,6 +3,58 @@ title: Release Notes
 weight: 8
 ---
 
+## Version 0.13.0
+
+New Features:
+
+- CRD support! A long awaited feature, MetalLB is now configurable via CRs. As a side effect of this new configuration,
+  it's now possible to choose to advertise addresses in L2 mode, BGP mode, both or just allocate the IP without advertising it.
+  On top of that, validating webhooks will ensure the validity of the configuration upfront, without needing to check the logs.
+  ([PR #1237](https://github.com/metallb/metallb/pull/1237), [PR #1245](https://github.com/metallb/metallb/pull/1245))
+  Please check the docs on how to use the converter tool to generate the CRs from your configmap.
+
+- Announcement node selector. It's possible to choose which nodes to advertise from the IPs coming from a given pool ([PR #1302](https://github.com/metallb/metallb/pull/1302))
+
+- BGPPeer selector. For any IP allocated from a given IPAddressPool, it is possible to choose the subset of BGPPeers
+  we want to advertise that IP to ([PR #1171](https://github.com/metallb/metallb/pull/1171)).
+
+- Kustomize configuration overlays. We now provide various overlays that implement different configuration degrees (as opposed to having)
+  one single manifest. ([PR #1254](https://github.com/metallb/metallb/pull/1254))
+
+- It's now possible to store BGP passwords as secrets (as an alternative to plain text passwords). ([PR #1264](https://github.com/metallb/metallb/pull/1264)).
+
+- LoadBalancerClass support: it's possible to have MetalLB listen only to services with the provided load balancer class to comply with [kubernetes loadbalancer class](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class). ([PR 1417](https://github.com/metallb/metallb/pull/1417)).
+
+Changes in behavior:
+
+- the biggest change is the introduction of CRDs and removing support for the configuration via ConfigMap. In order to ease the transition
+  to the new configuration, we provide a conversion tool from ConfigMap to resources (TODO Add link here)
+
+- the internal architecture was radically changed in order to accomodate CRDs, so please do not hesitate to [file an issue](https://github.com/metallb/metallb/issues).
+
+- The `AvoidBuggyIPs` flag was removed in order to reduce the api surface a bit. The same result can be achieved using ranges of IPs instead of
+  the CIDR annotation.
+
+- The metallb images from dockerhub are **deprecated**. From this release, only the images on quay.io will be supported and updated.
```

Comment:
A direct reference to the MetalLB organization listing all the container images would be best. Consider adding a link to https://quay.io/organization/metallb

Reply:
done